src/components/TvDetails/Season/index.tsx

```diff
@@ -29,34 +30,38 @@ const Season = ({ seasonNumber, tvId }: SeasonProps) => {
 
   return (
     <div className="flex flex-col justify-center divide-y divide-gray-700">
-      {data.episodes
-        .slice()
-        .reverse()
-        .map((episode) => {
-          return (
-            <div
-              className="flex flex-col space-y-4 py-4 xl:flex-row xl:space-y-4 xl:space-x-4"
-              key={`season-${seasonNumber}-episode-${episode.episodeNumber}`}
-            >
-              <div className="flex-1">
-                <div className="flex flex-col space-y-2 xl:flex-row xl:items-center xl:space-y-0 xl:space-x-2">
-                  <h3 className="text-lg">{episode.name}</h3>
-                  {episode.airDate && (
-                    <AirDateBadge airDate={episode.airDate} />
-                  )}
+      {data.episodeCount === 0 ? (
```

Comment:
Is it possible for the episode count to be nonzero but to have no episode data? I haven't looked into this, just wondering if that might be the case for announced upcoming seasons.

Reply:
Edit: See following comment. Should check length of `episodes` here instead.